srcpkgs/PrismLauncher/files/README.voidlinux

```diff
@@ -4,3 +4,6 @@ Void-specific instructions:
 You may not have the correct version of java installed for running specific
 versions of Minecraft. For Minecraft version 1.17 and up install openjdk17-jre
 and for older ones openjdk8-jre.
+
+If you get an error message that includes "org.lwjgl.opengl.LinuxDisplay.getAvailableDisplayModes",
+you may need to install xrandr.
```

Comment:
Would it be possible to catch this warning and give the user a better one? That wouldn't be the job of packaging, but we could report it upstream.